doc/wg/core/notes/core-notes-2024-09-20.md

```diff
@@ -0,0 +1,194 @@
+# Tock Meeting Notes 2024-09-20
+
+## Attendees
+
+- Alexandru Radovici
+- Alyssa Haroldsen
+- Amit Levy
+- Ben Prevor
+- Brad Campbell
+- Branden Ghena
+- Chris Frantz
+- Hudson Ayers
+- Johnathan Van Why
+- Kat Watson
+- Lawrence Esswood
+- Leon Schuermann
+- Pat Pannuto
+
+
+## Updates
+- Leon: Glueing together the GitHub integration to treadmill. I think we can get
+  an initial workflow merged that can run a job on every PR. Things will break
+  so I wouldn't make it a required check but we can go from there.
+- Leon: I was at the open source firmware conference, and I spoke to lowRISC,
+  and they're interested in treadmill for internal testing as well.
+- Amit: There is a nasty libtock-c timer bug in the unwrapping logic that
+  resulted in alarms getting lost. Sent a PR (libtock-c 466) with a fix. Would
+  be good to create a test.
+
+## VeeR EL2 PR (#4118) check-in
+- Branden: Been here for a little bit. It's changing the obsolete Swerv into the
+  new verilator simulation target veer el2. For the most part it's a rename plus
+  there are lines of code changing in chip-specific places.
+- Brad: Did they ever document it? They linked to a README but they delete their
+  READMEs so it doesn't mean much.
+- Branden: Yes, there is now documentation on how to run it and such.
+- Amit: It hadn't been clear to us what the relationship betwene swerv and veer
+  is. We were waiting to understand whether veer should be separate or replace
+  it, and now it's been answered.
+- Branden: There are chip-specific files like `io.rs`. Are we okay with that as
+  long as they're self-contained to the chip?
```

Comment:
```suggestion
- Branden: There are chip-specific files like `io.rs`. That probably none of us are able to truly review since we don't know this chip. Are we okay with that as
  long as they're self-contained to the chip?
```